scripts/relayCompiler.js

```diff
@@ -0,0 +1,91 @@
+require('dotenv/config')
+
+const chalk = require('chalk')
+const fs = require('fs')
+const path = require('path')
+const spawn = require('child_process').spawn
+const { execSync } = require('child_process')
+
+ensureRequirementsMet()
+
+// prettier-ignore
+const args = [
+  '--extensions', 'js', 'jsx', 'ts', 'tsx',
+  '--schema', path.resolve(process.cwd(), './data/schema.graphql'),
+  '--language', 'typescript',
+  '--src', path.resolve(process.cwd(), '../'),
+  '--artifactDirectory', path.resolve(process.cwd(), 'src/__generated__'),
+  '--include',
+      'force/src/**',
+      'reaction/src/**',
+  '--exclude',
+      'reaction/node_modules/**',
+];
+
+if (process.argv.includes('--watch')) {
+  args.push('--watch')
+}
+
+// TODO: We don't need this as we're cloning projects side-by-side and setting
+// relay --src root one level below project, where reaction lives.
+
+// Use this flag if linking node modules
+// if (process.argv.includes('--no-watchman')) {
+//   args.push('--no-watchman')
+// }
+
+const proc = spawn(
+  path.resolve(__dirname, '../node_modules/.bin/relay-compiler'),
+  args,
+  { stdio: 'inherit' }
+)
+
+proc.on('close', code => {
+  process.exit(code)
+})
+
+function ensureRequirementsMet() {
+  const requirements = [
+    {
+      test: () => '../../reaction',
+      log: () => {
+        console.log(
+          chalk.red('[scripts/relayCompiler] ERROR:'),
+          chalk.white(
+            'Cannot find local copy of Reaction, which must be cloned and',
+            'setup alongside Force. See https://github.com/artsy/reaction for',
+            'setup instructions.'
+          )
```

Comment:
Could we use git submodules instead of cloning next to the Force repo?

Reply:
This will be resolved when lib code is moved into reaction

Reply:
@alloy - i've been thinking about this and I feel like it would be good to leave relay tooling in here to keep things totally flexible across projects. 

Would it be possible to argue against using submodules? Most devs will have things checked out side by side and it would be nice to keep this really simple / straightforward. This is particularly important as we'll be using reaction's `yarn watch` feature and we don't want to have version mismatches between submodule and main project. (Speaking of which, I can't wait to remove the `--submodules` flag over at reaction and replace it with `graphql-cli`. I've been tripped up by that flag so many times, and I know a lot of other devs have too.)

Reply:
What is the part that you mean by Relay tooling?

I’m fine with not using git submodules 👍

Reply:
Oh, I wasn’t ever really thinking of that, just sibling clone vs submodule.